modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PlacedCommerceOrderShipmentFragmentTableFDSView.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.order.content.web.internal.frontend.data.set.view.table;
+
+import com.liferay.commerce.order.content.web.internal.constants.CommerceOrderFragmentFDSNames;
+import com.liferay.frontend.data.set.view.FDSView;
+import com.liferay.frontend.data.set.view.table.BaseTableFDSView;
+import com.liferay.frontend.data.set.view.table.FDSTableSchema;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilder;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilderFactory;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+@Component(
+	property = "frontend.data.set.name=" + CommerceOrderFragmentFDSNames.PLACED_ORDER_SHIPMENTS,
+	service = FDSView.class
+)
+public class PlacedCommerceOrderShipmentFragmentTableFDSView
+	extends BaseTableFDSView {
+
+	@Override
+	public FDSTableSchema getFDSTableSchema(Locale locale) {
+		FDSTableSchemaBuilder fdsTableSchemaBuilder =
+			_fdsTableSchemaBuilderFactory.create();
+
+		return fdsTableSchemaBuilder.add(
+			"id", "shipment-id",
+			fdsTableSchemaField -> fdsTableSchemaField.setSortable(true)
+		).add(
+			"createDate", "create-date",
+			fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
+				"dateTime"
+			).setSortable(
+				true
+			)
+		).add(
+			"expectedDate", "delivery-date",
+			fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
+				"dateTime"
+			).setSortable(
+				true
+			)
+		).add(
+			"oneLineAddress", "sent-to"
+		).add(
+			"itemsCount", "items"
+		).add(
+			"carrier", "carrier",
+			fdsTableSchemaField -> fdsTableSchemaField.setSortable(true)
```

Comment:
We need to report of malfunctioning of sorting and filters in general.

Reply:
Thread already opened on Slack